app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+  has_many :items
+
+  def active_model_serializer
+    UserSerializer
```

Comment:
This is only necessary to specify if the name does not follow the convention.  You should be safe to delete these lines for all your models